lib/paths.js

```diff
@@ -0,0 +1,8 @@
+const paths = {
```

Comment:
this name (or the file name) might be confusing or rather imprecise.

only when looking at the values themselves it might become clear that it's svg paths. if this file is closely coupled to your svg component, this file might belong there instead. that would also clarify the name.